test/gridded_data.cc

```diff
@@ -31,44 +35,52 @@ TEST(GriddedData, Basic) {
   // because of the pattern above we should end up with concentric circles
   // every ring should have all smaller rings inside it
   size_t rings = 0;
-  for (auto collection = std::next(contours.rbegin()); collection != contours.rend(); ++collection) {
-    // nothing here
-    auto& contour = collection->second.front();
-    if (contour.empty())
-      continue;
+  for (auto collection = contours.rbegin(); collection != contours.rend(); ++collection) {
+    auto& contour = collection->front();
+    ASSERT_FALSE(contour.empty());
     ++rings;
+    // skip the first (smallest ring)
+    if (collection == contours.rbegin())
+      continue;
     // if this is a ring the iso lines with lesser units should be contained within it
-    for (const auto& p : std::prev(collection)->second.front().front()) {
-      ASSERT_TRUE(p.WithinPolygon(contour.front()))
-          << "Ring " + std::to_string(collection->first) + " should contain ring " +
-                 std::to_string(std::prev(collection)->first);
+    for (const auto& p : std::prev(collection)->front().front()) {
+      ASSERT_TRUE(p.WithinPolygon(contour.front())) << "Ring should contain smaller ring";
     }
   }
 
   // there should be quite a few rings here
-  ASSERT_NE(rings, 0) << "There should be at least a few rings here";
-
+  ASSERT_EQ(rings, 6) << "There should be at least a few rings here";
   /*
+  auto iso_marker = iso_markers.begin();
   std::cout << "{\"type\":\"FeatureCollection\",\"features\":[";
-  for(const auto& feature : contours) {
-    std::cout << "{\"type\":\"Feature\",\"properties\":{\"iso\": " << feature.first << "},";
-    std::cout << "\"geometry\":{\"type\":\"MultiLineString\",\"coordinates\":[";
-    for(const auto& line : feature.second) {
-      std::cout << "[";
-      for(const auto& coord : line) {
-        std::cout << "[" << coord.first << "," << coord.second << "]" << (&coord != &line.back() ?
-  "," : "");
+  for (const auto& feature_collection : contours) {
```

Comment:
this is still useful for debugging but it looked like it hadnt been updated for a long time (probably before the first implementation was finished). so i updated it to work with the current implementation